### PR TITLE
additions

### PR DESCRIPTION
**Pull request by rawb0t**

### CODE REVIEW

File: _data/retail.yml [hunk 1]
```diff
@@ -290,6 +297,14 @@ websites:
       facebook: northerntool
       email_address: intlenglishsales@northerntool.com
 
+    - name: Omura
+      url: https://shopomura.com
+      twitter: shop_omura
```
Comment (kenman345) [inline _data/retail.yml]: img tag....

Comment (t0p0lin0) [reply _data/retail.yml]: I suppose `img` could be added until we confirm that it is no longer required, since we have our `generic.png` idea in place.

Comment (kenman345) [reply _data/retail.yml]: he included an image though....

Comment (t0p0lin0) [reply _data/retail.yml]: My bad. In that sense, yes, it should be added. Appears I am not the best reviewer of YAML, but I will keep trying!

Comment (rawb0t) [reply _data/retail.yml]: whoops. fixed.

**Pull request merged**